app/scripts/migrations/089.ts

```diff
@@ -73,12 +74,7 @@ function transformState(state: Record<string, unknown>) {
       ),
     );
   } else if (!isObject(state.NetworkController.providerConfig)) {
-    global.sentry?.captureException?.(
```

Comment:
This sentry call was added so that we could investigate the cases where `state.NetworkController.providerConfig` is `undefined`. I don't think we should convert this to a `log.warn` unless either:
(A) We know why `state.NetworkController.providerConfig` is undefined, or
(B) We have concluded that we will never understand why, or that we have already gathered all the information we might possibly get